Makefile

```diff
@@ -483,4 +486,20 @@ verify-modules: modules  ## Verify go modules are up to date
 help: ## Show this help.
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
+space: 
+	$(MAKE) -C space-framework
+.PHONY: space
+
+space-crds: 
+	$(MAKE) -C space-framework crds
+.PHONY: space-crds
+
+space-codegen: 
+	$(MAKE) -C space-framework codegen
+.PHONY: space-codegen
+
+space-modules: 
+	$(MAKE) -C space-framework modules
+.PHONY: space-modules
```

Comment:
Why do we want targets in this Makefile that are just aliases for targets in another Makefile?  Why not tell contributors to use the underlying Makefile and targets directly?

Reply:
Agree